src/App.js

```diff
@@ -158,6 +155,10 @@ class Root extends React.Component {
    * @param {boolean} data Boolean if websocket is online
    */
   isOnlineUpdate = (data) => {
+    if (data) {
+      // Check api version everytime we connect to the server
+      version.checkApiVersion();
```

Comment:
On Hathor Mobile Wallet, we check the API version before marking the connection as online. Should we do the same here?

Reply:
The reason I remove this from `addressesLoaddedUpdate` is that we don't need to check the api version every time we get a new tx. In my tests, I had 400 addresses with 1 tx each. During wallet load, we'd call `checkApiVersion` 400 times in a short period.

We could change it to mark as online only after this check, but I didn't want to change the current behavior. I'm not sure if there'd be any other side effects of doing this. To be honest, this is not needed to fix the bug. It just seemed something simple that could improve load, without incurring any other changes.

Reply:
I agree. Thanks!